auth-web/src/util/constants.ts

```diff
@@ -247,6 +247,48 @@ export enum NrEntityType {
     INFO = 'INFO', // special value for sub-menu
 }
 
+export enum NrRequestType {
+  // Name Change
+  BEC = 'Name Change',
+  CCC = 'Name Change',
+  CCP = 'Name Change',
+  CCR = 'Name Change',
+  CFR = 'Name Change',
+  CUL = 'Name Change',
+
+  // Continuation in
+  BECT = 'Continuation In',
+  CCCT = 'Continuation In',
+  CTC = 'Continuation In',
+  CT = 'Continuation In',
+  ULCT = 'Continuation In',
+
+  // Alteration
+  BECV = 'Alteration',
+  CCV = 'Alteration',
+  BECR = 'Alteration',
+  UC = 'Alteration',
+
+  // Incorporation
+  BC = 'Incorporation', // BC Benefit Company Incorporation
+  CC = 'Incorporation', // CCC - Incorporation/Amalgamation
+  CP = 'Incorporation', // Cooperative - Incorporation/Amalgamation
+  CR = 'Incorporation', // BC Company - Incorporation/Amalgamation
+
+  // Registration
+  FR = 'Registration', // Sole Proprietorship/General Partnership/DBA - Registration
+  UL = 'Registration', // Unlimited Liability Company
+  XCR = 'Registration', // Corporation (Foreign) - Extrapro Registration/Amalgamation
```

Comment:
I think FR ("firm") is the only registration here. The others are incorporations.